browser/src/react-redux/App/User/User.action.js

```diff
@@ -1,8 +1,8 @@
 const setUser = (user) => {
```

Comment:
Because of the reason in this comment:
https://github.com/donthedeveloper/site-builder/pull/82#discussion_r255308416

Let's go ahead and change the name of this file from `User.action.js` to `User.actions.js`. This will contain all potential actions for our user authentication.